### ISSUE TITLE
juxt.clip.core/ref no longer expanded in 0.22.0

### ISSUE THREAD

**Issue by apa512** [SevereOverfl0w/high#21]
After updating from 0.21.0 to 0.22.0, juxt.clip.core/ref is no longer replaced by the started component.

config.edn:
```clj
{:components
 {:a {:start 1}
  :b {:start (prn ">>>" (juxt.clip.core/ref :a) (type (juxt.clip.core/ref :a))}}
```

Code:
```clj
(clip.core/start (aero.core/read-config (clojure.java.io/resource "config.edn")))
```

In 0.22.0:
```clj
">>>" (clip/ref :a) clojure.lang.PersistentList
```

In 0.21.0:
```clj
">>>" 1 java.lang.Long
```

Am I correct in thinking that 0.21.1 is the expected behavior? Or have I been doing something weird that just happened to work in earlier versions?

**Comment by SevereOverfl0w** [comment]
Broken in 5b344d94aff56c2f57a8f7088af0c212e047ea14

**Comment by SevereOverfl0w** [comment]
Fixed in 0.23.0

**Comment by apa512** [comment]
Thank you
